ddtrace/tracer/option.go

```diff
@@ -102,6 +104,9 @@ func newConfig(opts ...StartOption) *config {
 	}
 	c.dogstatsdAddr = net.JoinHostPort(statsdHost, statsdPort)
 
+	if v := os.Getenv("DD_TRACE_ANALYTICS_ENABLED"); v != "" {
```

Comment:
Are we sure we want to accept any value as turning it on?
I think `DD_TRACE_ANALYTICS_ENABLED=false` causing it to be enabled is counter-intuitive.

Maybe we can parse it as a boolean.

Reply:
You’re right. It was a hasty copy pasta. I should use the `boolEnv` you have in your PR.

Reply:
`boolEnv` https://github.com/DataDog/dd-trace-go/blob/v1/ddtrace/tracer/option.go#L497 is merged now, if you want to use it. It's not critical, though.